src/Lms/Canvas/CanvasLms.php

```diff
@@ -51,7 +51,7 @@ public function __construct(
         UtilityService $util,
         Security $security,
         SessionService $sessionService,
-        HtmlService $html)
+        HtmlService $html) 
```

Comment:
These trailing spaces are just creating noise in the diff; would you mind removing them?